### ISSUE TITLE
Ability to rebuild from list page

### ISSUE THREAD

**Issue by sgringwe** [buildkite/feedback#191]
Sometimes I need to cancel from the list page, and it'd be great to have the ability to rebuild from that page as well. Even better would be "Cancel and Rebuild" option as one button.

**Comment by keithpitt** [comment]
Interesting idea @sgringwe! I'm keen to hear more about the situation you're in when you need to do that if you can talk it through a bit more?

**Comment by schickling** [comment]
Yeah, it would be nice to cancel + rebuild builds from the list page.

**Comment by keithpitt** [comment]
We actually used to have it like this! (cancel + rebuild builds from the list page) but I can't quite remember why we removed it. I'll have another look at it in the new year and see if we can change it back! Thanks for reminding us about it @schickling!
